spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "As a visitor when I visit '/ingredients'" do
+  before :each do
+    @gb = Ingredient.create!(name: "Ground Beef", cost: 2)
+    @nacl = Ingredient.create!(name: "Salt", cost: 4)
+    @avo = Ingredient.create!(name: "Avocado", cost: 3)
+  end
+
+  it "I see a list of all the ingredients including their name and cost" do
+    visit "/ingredients"
+
+    expect(page).to have_content("#{@gb.name}: #{@gb.cost}")
```

Comment:
Nice work with these assertions that anchor the ingredient cost to their name -- with this assertion we can be sure that the `2` is showing up next to `Ground Beef` and not `Salt`